### ISSUE TITLE
batch node documentation omission

### ISSUE THREAD

**Issue by andysc** [node-red/node-red#1601]
v0.18.0

info tab documentation for the batch node omits to mention that it needs a "parts" part to make it work

**Comment by dceejay** [comment]
errr - that's because it doesn't.... 
It is the one that adds the ports for you...

**Comment by andysc** [comment]
oh, sorry - I missed a sentence off... when you select "concatenate sequences".
```
[{"id":"cb3e9907.5ae408","type":"inject","z":"6d47234b.61382c","name":"","topic":"a","payload":"","payloadType":"date","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":230,"y":260,"wires":[["b733b868.23d748"]]},{"id":"b733b868.23d748","type":"batch","z":"6d47234b.61382c","name":"","mode":"concat","count":10,"overlap":0,"interval":10,"allowEmptySequence":false,"topics":[{"topic":"a"},{"topic":"b"}],"x":390,"y":280,"wires":[["650c41ef.1a498"]]},{"id":"4b4c59ba.0bb428","type":"inject","z":"6d47234b.61382c","name":"","topic":"b","payload":"","payloadType":"date","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":230,"y":300,"wires":[["b733b868.23d748"]]},{"id":"650c41ef.1a498","type":"debug","z":"6d47234b.61382c","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"false","x":558,"y":278,"wires":[]}]
```

**Comment by knolleary** [comment]
Fix pushed 6f69995f4ea6522a092d3ffb1007654a6e7b0bae
